src/hooks/use-field-id/use-field-id.spec.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render } from '../../test-utils';
```

Comment:
You know that there is a testing library version for testing hooks?

https://github.com/testing-library/react-hooks-testing-library

Reply:
Hm, but doesn't this break the test-library philosophy of testing like a user? A user doesn't render hooks 🤔

Reply:
I think I would rather add this in a separate PR

Reply:
I looked at `react-hooks-testing-library` too when implementing the `useHubSpotIdentity`-hook. I decided against it cause it's testing hooks more like a unit. However, giving the "advantage" of not having to "wrap" a hook into a `TestComponent`. I didn't mind the additional setup though for the benefit of not testing the fact that something is a hook.

Reply:
Well, the "user test" should happen in the component right? The hook is just a function, which can be simply unit tested.

Reply:
Not really, what's the point of testing the hook in every single component that uses it? It makes sense for me to test is as a user once.

Reply:
BTW, I'm just mentioning this because it's come up in the past that we misunderstand each other / get into arguments.

Saying "You know that there is a testing library version for testing hooks?" comes across as rude in English. I don't think this is your intention, but that's my first impression upon reading your comment as a native speaker...

Reply:
Yes of course this wasn't my intention, I wanted to make you aware that there was this library for testing hooks specifically. Just that.

As for the approach, it's fine either way. Thanks